src/components/TodoComponents/Todo.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import './Todo.css';
+
+class ToDo extends React.Component{
+    constructor(){
+        super();
+
+        this.state = {
```

Comment:
I would highly recommend having this state info inside the App component, subsequently you would pass the state from the App component to the TodoList component, with that info you would make a new ToDo with each element inside the array.